SwiftMessageBar/SwiftMessageBar.swift

```diff
@@ -121,9 +121,11 @@ public final class SwiftMessageBar {
   /// - Returns: A UUID for the message. Can be used to cancel the display of a specific message
   public static func showMessageWithTitle(_ title: String? = nil, message: String? = nil, type: MessageType,
                                           duration: TimeInterval = 3, dismiss: Bool = true,
-                                          callback: Callback? = nil) -> UUID {
+                                          languageDirection: NSLocale.LanguageDirection = .leftToRight,
```

Comment:
I committed the default direction change. I think the name is fine but if you feel it should be renamed then you can change.

I am facing one issue. You merged the last commit and i can see the pod version 3.0.2 in podspec and other changes on your repo. But when i update the pod or even i tried to install in a fresh project, its still fetching the old version 3.0.1. I don't why this is happening. Can you  please look in to it.